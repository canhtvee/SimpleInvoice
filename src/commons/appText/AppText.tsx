import React from 'react';
import {Text, TextProps} from 'react-native';
import {Colors, Sizes} from '../../utils';

export function AppText({children, style, ...props}: TextProps) {
  return (
    <Text
      style={[{fontSize: Sizes.regular, color: Colors.text}, style]}
      {...props}>
      {children}
    </Text>
  );
}
