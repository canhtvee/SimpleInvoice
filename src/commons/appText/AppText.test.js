import React from 'react';

import {cleanup} from '@testing-library/react-native';
import {create} from 'react-test-renderer';

import {AppText} from './AppText';

/**
 * Run test command
 * yarn test -u -t="AppText"
 */

afterEach(cleanup);

describe('AppText', () => {
  test('should render correctly', () => {
    const tree = create(<AppText />);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
