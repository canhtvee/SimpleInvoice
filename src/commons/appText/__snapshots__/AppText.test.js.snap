// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppText should render correctly 1`] = `
<Text
  style={
    [
      {
        "color": "#000000",
        "fontSize": 27.5,
      },
      undefined,
    ]
  }
/>
`;
